src/build/__init__.py

```diff
@@ -201,6 +201,7 @@ def build(self, distribution, outdir):  # type: (str, str) -> None
         :param outdir: Outpur directory
         '''
         build = getattr(self.hook, 'build_{}'.format(distribution))
+        outdir = os.path.realpath(outdir)
```

Comment:
Should this be `realpath` or `abspath`?

Not sure what the relative merits are of resolving vs. not resolving symbolic links.

Reply:
Not sure. I went with `realpath` because Windows has given me some trouble with symlinks in the past, I figured it didn't matter that much anyway.

Reply:
Do you want me to change to `abspath`?

Reply:
Hard to tell which is better or what the failure modes are — I wasn't really sure which is better and thought I'd flag it for consideration.

I don't think it will be that big a deal if we get it wrong though. Either someone will complain because something broke in some configuration that involves symlinks or they'll complain about something like the path being recorded incorrectly or something. I suppose we'll find out!

Reply:
Yeah, I changed it to `abspath`, we can merge that and see if anyone complains. It's a low risk change anyway, I think.